R/linear_pool_sample.R

```diff
@@ -21,18 +21,17 @@ linear_pool_sample <- function(model_out_tbl, weights = NULL,
                                weights_col_name = "weight",
                                model_id = "hub-ensemble",
                                task_id_cols = NULL,
+                               comp_unit_cols = NULL,
                                n_output_samples = NULL) {
 
```

Comment:
Question about this function: are we doing something in here to ensure that `output_type_id`s from different models are distinct in the ensemble?  It may be in there, I just didn't see it.